python/fledge/services/core/server.py

```diff
@@ -1728,10 +1742,13 @@ async def restart(cls, request):
                 sys.argv.append('')
 
             python3 = sys.executable
-            os.execl(python3, python3, *sys.argv)
+            os.execl(python3, python3, *sys.argv)  # Replaces current process, no return
 
-            return web.json_response({'message': 'Fledge stopped successfully. '
```

Comment:
What was it doing earlier?

Reply:
NOTHING! As it replaces current process, no return

Reply:
`restart` def is bound to endpoint, We must return some response to caller.

Reply:
It is there
```
$ curl -sX PUT localhost:8081/fledge/restart | jq
{
  "message": "Fledge restart has been scheduled."
}
```